lib/bolt/project.rb

```diff
@@ -50,6 +50,20 @@ def self.find_boltdir(dir, logs = [])
     def self.create_project(path, type = 'option', logs = [])
       fullpath = Pathname.new(path).expand_path
 
+      if %w[user system].include?(type)
```

Comment:
We shouldn't try to create the system level project here. It's almost guaranteed to fail and it's not really desirable anyway, since a system level directory shouldn't be owned by an arbitrary user.